src/pages/CogPage.vue

```diff
@@ -85,9 +96,33 @@ export default class CogPage extends Vue {
           return;
```

Comment:
the whole "related" thing won't be called if cogs were already loaded due to this return (e.g., the user opened a cog page from the home page or wherever else where cogs are pre-loaded in full).